source/containers.tex

```diff
@@ -14895,8 +14895,8 @@
       { x.swap(y); }
 
   private:
-    containers c;               // \expos
-    key_compare compare;        // \expos
+    containers \exposid{c};               // \expos
```

Comment:
There was one thing I wasn't sure how to do markup-wise, so I left it alone / out-of-scope: Line 14978 has
```
Initializes \tcode{c.keys} with \tcode{std::move(key_cont)}
```
where ideally the `c` should be expos-only but the `keys` shouldn't.

Reply:
This is what I meant with https://github.com/cplusplus/draft/pull/5914#issuecomment-1291043847. By convention, marking with `\exposid` in the synopsis is not enough. All uses of those members throughout are marked with `\exposid`.

Reply:
I think you can just use
```
Initializes \tcode{\exposid{c}.keys} with \tcode{std::move(key_cont)}
```